oxd-common/src/main/java/org/gluu/oxd/common/ErrorResponse.java

```diff
@@ -4,17 +4,18 @@
 package org.gluu.oxd.common;
 
 import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
```

Comment:
class must not contain references to `org.codehaus.jackson`. It should be `com.fasterxml.jackson`. Same for `JsonNode` above.